src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -32,6 +32,948 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+                var publication1Releases = publicationViewModels[0].Releases;
+                Assert.Equal(4, publication1Releases.Count);
+                Assert.Equal(publication1Release1.Id, publication1Releases[0].Id);
+                Assert.Equal(publication1Release2.Id, publication1Releases[1].Id);
+                Assert.Equal(publication1Release3.Id, publication1Releases[2].Id);
+                Assert.Equal(publication1Release4.Id, publication1Releases[3].Id);
+
+                Assert.Equal(publication2.Id, publicationViewModels[1].Id);
+                var publication2Releases = publicationViewModels[1].Releases;
+                Assert.Equal(2, publication2Releases.Count);
+                Assert.Equal(publication2Release1.Id, publication2Releases[0].Id);
+                Assert.Equal(publication2Release2.Id, publication2Releases[1].Id);
+            }
+        }
+
+        [Fact]
+        public async Task GetMyPublicationsAndReleasesByTopic_CanViewAllReleases_MethodologyOrder()
+        {
+            var topic = new Topic
+            {
+                Id = Guid.NewGuid(),
+            };
+
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+                ReleaseName = "2001",
+                TimePeriodCoverage = TimeIdentifier.Week1,
+            };
+            var methodology1Version1 = new MethodologyVersion
+            {
+                Id = Guid.NewGuid(),
+                AlternativeTitle = "Methodology 1 Version 1",
+                Version = 0,
+                Status = Draft
+            };
+
+            var methodology2Version1 = new MethodologyVersion
+            {
+                Id = Guid.NewGuid(),
+                AlternativeTitle = "Methodology 2 Version 1",
+                Version = 0,
+                Status = Approved
+            };
+
+            var methodology2Version2 = new MethodologyVersion
+            {
+                Id = Guid.NewGuid(),
+                AlternativeTitle = "Methodology 2 Version 2",
+                Version = 1,
+                Status = Draft,
+                PreviousVersionId = methodology2Version1.Id
+            };
+
+            var methodology3Version1 = new MethodologyVersion
+            {
+                Id = Guid.NewGuid(),
+                AlternativeTitle = "Methodology 3 Version 1",
+                Version = 1,
+                Status = Approved,
+            };
+
+            var publication1 = new Publication
+            {
+                Title = "publication1",
+                Releases = new List<Release>
+                {
+                    release,
+                },
+                Methodologies = AsList(
+                    new PublicationMethodology
+                    {
+                        Methodology = new Methodology
+                        {
+                            Slug = "methodology-2-slug",
+                            Versions = AsList(methodology2Version2, methodology2Version1)
```

Comment:
Wherever we see "AsList", try to replace with "ListOf" from within the same util class!  We're trying to phase out the former in favour of the latter.